pygmt/src/grdvolume.py

```diff
@@ -74,7 +76,23 @@ def grdvolume(grid, output_type="pandas", outfile=None, **kwargs):
           ``outfile``)
         - :class:`pandas.DataFrame` or :class:`numpy.ndarray` if ``outfile``
           is not set (depends on ``output_type`` [Default is
-          class:`pandas.DataFrame`])
+          :class:`pandas.DataFrame`])
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # Load a grid of @earth_relief_30m data, with an x-range of 10 to 30
+    >>> # degrees, and a y-range of 15 to 25 degrees
+    >>> grid = pygmt.datasets.load_earth_relief(
+    ...     resolution="30m", region=[10, 30, 15, 25]
+    ... )
+    >>> # Create a numpy ndarray that contains the contour, area, volume, and
+    >>> # maximum mean height above the plane specified by the given contour
+    >>> # and below the surface; set the minimum contour z-value to 200, the
+    >>> # maximum to 400, and the interval to 50.
+    >>> output_array = pygmt.grdvolume(
+    ...     grid=grid, contour=[200, 400, 50], output_type="numpy"
```

Comment:
Why not use a `pandas` output type?

```suggestion
    >>> # Create a pandas dataframe that contains the contour, area, volume,
    >>> # and maximum mean height above the plane specified by the given
    >>> # contour and below the surface; set the minimum contour z-value to
    >>> # 200, the maximum to 400, and the interval to 50.
    >>> output_dataframe = pygmt.grdvolume(
    ...     grid=grid, contour=[200, 400, 50], output_type="pandas"
```